src/server/index.jsx

```diff
@@ -292,6 +295,7 @@ server.get(
           requestUrl: url,
         });
 
+        console.error(error);
```

Comment:
I needed this to help me debug errors when implementing view tracking, and it's kinda useful, so would like to keep it 😸